ups/sims_alertsim.table

```diff
@@ -1,12 +1,104 @@
-setupRequired(sims_utils)
-setupRequired(sims_catalogs)
-setupRequired(sims_catUtils)
-setupRequired(python)
-setupRequired(pymssql)
-setupRequired(daf_persistence)
-setupRequired(astropy)
-
-#This is needed by the mssql python module
+if (type == exact) {
```

Comment:
I think the `if (type==exact)` form is automatically generated by EUPS at install time.  If you look at even the fundamental DM packages, for example:

https://github.com/lsst/afw/blob/master/ups/afw.table
https://github.com/lsst/obs_lsstSim/blob/master/ups/obs_lsstSim.table

the table files look more like what used to be in this file than what is in there now.

Is there a specific reason you made this change?

Reply:
No, that was my bad.